packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,28 @@
+import { SecuritySchemeElement } from '@swagger-api/apidom-ns-openapi-3-1';
+import { AnnotationElement } from '@swagger-api/apidom-core';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const securitySchemeTypeRefractorPlugin =
+  ({ annotations }: SecuritySchemeTypePluginOptions) =>
+  () => ({
+    visitor: {
+      SecuritySchemeElement(element: SecuritySchemeElement) {
+        if (element.get('type')?.toValue() === 'mutualTLS') {
+          const annotation = new AnnotationElement(
+            '"mutualTLS" type is not supported in OpenAPI 3.0.3.',
+            { classes: ['warning'] },
+            { code: 'mutualTLS' },
+          );
+
+          annotations.push(annotation);
+        }
+
+        return element;
```

Comment:
If we don't want to do nothing with the element, we're just return `undefined`. More info in https://github.com/swagger-api/apidom/blob/main/packages/apidom-ast/src/traversal/visitor.ts#L144

```suggestion
        return undefined;
```